README.md

```diff
@@ -334,6 +347,19 @@ stockfish.will_move_be_a_capture("e5d6")  # returns Stockfish.Capture.EN_PASSANT
 stockfish.will_move_be_a_capture("f1e2")  # returns Stockfish.Capture.NO_CAPTURE  
 ```
 
+### Convert human-style notation into the notation Stockfish uses
```

Comment:
perhaps add some more examples. `d5` and `xd5` are valid as well for example. also `ooo`,etc. Also I'm no expert in chess notation. I just took what I saw on lichess and derived the rules for how the notation works for myself. It might be missing something or be inaccurate